@xen-orchestra/lite/src/components/pool/dashboard/PoolDashboardTasks.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <UiCard>
+    <UiCardTitle :count="pendingTasks.length">{{ $t("tasks") }}</UiCardTitle>
+    <TasksTable :pending-tasks="pendingTasks" />
```

Comment:
We don't show the finished tasks on the dashboard? Is it on purpose?

Reply:
Yes, finished tasks are only displayed on Tasks tab.

Tell me if you think that they should be displayed on Dashboard too.

Reply:
No, I think it makes sense, I just wanted to make sure it was on purpose.